src/Cake.Npm/Install/NpmInstallSettingsExtensions.cs

```diff
@@ -106,6 +106,53 @@ public static NpmInstallSettings ForProduction(this NpmInstallSettings settings)
         }
 
         /// <summary>
+        /// Defines that npm should npm should prevent optional dependencies from being installed
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmInstallSettings.NoOptional"/> set to <c>true</c>.</returns>
+        public static NpmInstallSettings WithNoOptional(this NpmInstallSettings settings)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            return settings.WithNoOptional(true);
+        }
+
+        /// <summary>
+        /// Defines that npm should not prevent optional dependencies from being installed
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmInstallSettings.NoOptional"/> set to <c>false</c>.</returns>
+        public static NpmInstallSettings WithoutNoOptional(this NpmInstallSettings settings)
```

Comment:
`WithoutNoOption` is a double negative. IMHO `WithOptionalDependencies` would be clearer

Reply:
Thanks for the feedback, greatly appreciated. I've adjusted as per your suggestions. I've leave the underlying "WithNoOptional" method to reflect the actual NPM setting, but I agree it reads much nicer without the double negative!